### DISCUSSION TITLE
Add more control like Contrast and R | G | B

### DISCUSSION THREAD

**Discussion by CrazyThunderbird** [xanderfrangos/twinkle-tray#402] [Ideas]
I would really like if there would be the implementation of Contrast aswells the RGB sliders aswell.

Actually i use ClickMonitorDDC (which isn't continued afaik) well it does the job pretty well but i really like the idea of twinkle tray

**Comment by xanderfrangos** [comment]
Hi @CrazyThunderbird,

The latest version of Twinkle Tray does support contrast control, but it does not support RGB adjustments. If you don't mind me asking. what do you use the RGB adjustments for?

**Comment by CrazyThunderbird** [comment]
I use this to make a "calibrated" darkmode, instead of the windows night mode. But also to control the rgb values when i calibrate my screens without to fiddle behind the screen with the buttons

But just because when i use lightroom my editing monitor switches automatically to 6500k 140cd/m² (you can link such things in ClickMonitorDDC.

Looks like this:
![image](https://user-images.githubusercontent.com/109760390/180438152-3d92ad4b-da88-4cdd-a41f-7d5e9696f519.png)


and those are my hotkeys, ignore DISPLAY4 and so on, this is when i used a capture card and the screens didnt get recognized properly by names anymore :D
![image](https://user-images.githubusercontent.com/109760390/180438105-2f90615d-dada-4e2f-9511-74b8040a3681.png)



Settings for the screens looking like this

> ASUSMG279 b 39 c 80 r 255 g 237 l 235 BenQEW3270U b 44 c 51 r 255 g 242 l 240 DELLU2713HM b 73 c 70 r 238 g 255 l 228

which i honestly don't really like, yeah it's in 255 steps, but real values would be doing it much better ^^
